### ISSUE TITLE
Theming LXDE panel

### ISSUE THREAD

**Issue by sabrehagen** [B00merang-Project/Windows-10#91]
Hi,

I've installed the dark theme successfully under LXDE using `transpack.sh` and my icons and window decorations look correct. However the panel along the bottom of the screen was not transformed like the one in your screenshot. What can I do to remedy this?

Thanks

**Comment by Elbullazul** [comment]
@sabrehagen screenshot please?

**Comment by sabrehagen** [comment]
I have since done my own hacks to fix this. It's not nice, but it's more effort to go back to the state I was in when I logged this issue. Sorry for not including a screenshot, I believe I intended to but maybe never posted the link. I'll reopen the issue if I start from a fresh install and encounter the same error.

Essentially, the 'start button' icon was not applied to the menu widget, leaving it with the original LXDE icon. The panel icon size was not increased (30px seems to be the windows size, they were at 12px or so), and the cortana and expose icons were not added to the panel.
